### DISCUSSION TITLE
Rank 30 Back to Rank 0

### DISCUSSION THREAD

**Discussion by ghost** [osm-search/Nominatim#2859] [General]
Hi, I'm currently trying to install my own instance of Nominatim (on E2 32vCPU and 128GB of RAM), after 3/4 days all 30 ranks were finished, my question is that after the rank 30 finished it went back to rank 0 and announces me another 17 hours, is that normal?

**Comment by mtmail** [comment]
That's normal. After 30 follows rank 0, then interpolation lines, then postcodes.

**Reply by ghost** [reply]
And then its all finished? (Thanks for the quick response by the way). And is there a time estimate for these steps?

**Reply by mtmail** [reply]
Not quite. Then follows 'Post-process tables' and 'Recompute word counts', each 2h. 2-3 days for a full planet installation is normal.

**Reply by ghost** [reply]
Okay thank you, also a bit off-topic question, after that the full download is finished, if I want to stop/restart the server how long does Nominatim take to start back up? Is it fast since everything is indexed?

**Reply by mtmail** [reply]
Import puts all data into Postgresql. The start is almost instant. Do you expect the postgresql data to be somehow deleted during a stop or restart?

**Reply by ghost** [reply]
Shouldn't be, I also remember reading to turn back o n fsync and full_page_writes after import. Thank you for your answers !
